### PR TITLE
Create digital test landing page > #53

### PR DESCRIPTION
**Pull request by thejoefriel**
#### Issues this PR relates to
#53 

---

#### Any info the reviewer needs to know to test 
e.g. Add env vars, run npm install

go to /digital-test

---

#### Any info the team needs to consider before merged into development
e.g. Will a database need to be rebuilt (and is this ok if existing data), add env vars

---

#### Quick Pull Request Checklist
Please make sure you ensure the following has been done before marking your PR as awaiting-review. For full checklist and process review, please refer to Process Guide (#8)
- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- The new work has been checked with Axe accessibility checker and doesn't throw significant errors  
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs

### CODE REVIEW

File: client/src/pages/TestLanding/index.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Row, Col } from '../../components/Grid';
+import * as T from '../../components/Typography';
+import * as S from './style';
+import * as R from '../../constants/nav-routes';
+import Icon from '../../components/Icon';
+import Button from '../../components/Button';
+
+const Landing = () => {
```
Comment (RamyAlshurafa) [inline client/src/pages/TestLanding/index.js:9]: make the name of the component matches the folder name, so its easier to find when using the react chrome extention

File: client/src/pages/TestLanding/style.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import styled from '@emotion/styled';
+import setMargin from '../../helpers/set-margin';
+
+export const ClockWrapper = styled.div`
+  ${setMargin}
```
Comment (RamyAlshurafa) [inline client/src/pages/TestLanding/style.js:5]:
```suggestion
  ${setMargin};
```

File: client/src/pages/TestLanding/index.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  return (
+    <article>
+      {/* <BackLink mb="5" /> */}
```
Comment (RamyAlshurafa) [inline client/src/pages/TestLanding/index.js:12]: do you need this?

File: client/src/App.js [hunk 4]
```diff
@@ -33,6 +35,16 @@ function App() {
               layout="notification"
               headerIcon="email"
             />
+            <Route
+              exact
+              path={R.TEST}
+              isLoggedIn={isLoggedIn}
+              isMounted={isMounted}
+              Component={() => <TestLanding onLogin={() => {}} />}
```
Comment (RamyAlshurafa) [inline client/src/App.js:43]:
```suggestion
              Component={TestLanding}
```

**Review by RamyAlshurafa — CHANGES_REQUESTED** [review]

File: client/src/pages/TestLanding/index.js (hunk 1 above)
Comment (thejoefriel) [reply client/src/pages/TestLanding/index.js:9]: done

File: client/src/pages/TestLanding/index.js (hunk 3 above)
Comment (thejoefriel) [reply client/src/pages/TestLanding/index.js:12]: deleted

File: client/src/pages/TestLanding/style.js (hunk 2 above)
Comment (thejoefriel) [reply client/src/pages/TestLanding/style.js:5]: done

File: client/src/App.js (hunk 4 above)
Comment (thejoefriel) [reply client/src/App.js]: fixed

**Review by RamyAlshurafa — APPROVED** [review]

**Pull request merged**